packages/xrpl-crypto/jest.config.js

```diff
@@ -0,0 +1,8 @@
+// Jest configuration for api
+const base = require('../../jest.config.base.js')
```

Comment:
@ckniffen Should this package use jest? (I vaguely remember you hoping to directly use other tools, but not sure if that's hard to do for this package)

Not sure if it matters much since there's not much to test yet, but wanted to call out in case there's something we should do.

Reply:
I wouldnt worry too much as long as it doesnt used advanced jest features that will interfere with running tests in the browser